assembly/src/docker/Dockerfile

```diff
@@ -15,31 +15,87 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
+FROM debian:bookworm-slim AS amq_tpl
```

Comment:
Is it not possible to use pure JDK base ?

Reply:
This stage of the build is only to prepare the default config. It does not require java at all.
The actual image still uses eclipse-temurin:17-jre as a base

Reply:
Ah yes, I see that now. Sorry, I was confused. Thanks for the details !